iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -36,6 +36,9 @@ public static class MessageSystemPropertyNames
         /// </summary>
         public const string To = "to";
 
+        /// <summary>
```

Comment:
For everything in this class, I got inputs from the IoT hub team.

Reply:
does DeliveryCount hold the max allowed value, or the current count of the no of times the message has transitioned between enqueued and invisible?
It might also help to add a link to the ms doc, so that users can visualize the entire lifecycle: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2d#the-cloud-to-device-message-life-cycle